### ISSUE TITLE
Collections

### ISSUE THREAD

**Issue by NicoLaval** [sdmx-twg/vtl#506]
Is `t := "A" in { "a", TEST };`, where test is a variable defined previously, correct?

That is, should collections only contain constants or can they also contain variables?

**Comment by NicoLaval** [comment]
According to G4 file:

```
lists:
    GLPAREN  constant (COMMA constant)*  GRPAREN
;

constant:
    INTEGER_CONSTANT
    | NUMBER_CONSTANT
    | BOOLEAN_CONSTANT
    | STRING_CONSTANT
    | NULL_CONSTANT
;
```

Values has to be `constant`. Is there a problem if we extend this definition to variables?

**Comment by hadrienk** [comment]
This should probably be changed to accept expressions indeed.

**Comment by antonio-olleros** [comment]
I agree!

**Comment by linardian** [comment]
I agree too! So please @NicoLaval fix the grammar to accept also set/valedomain identifiers. Thanks!

**Comment by NicoLaval** [comment]
Please see #537
